profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_dgt_connector/tmgmt_dgt_connector.module

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+ * @file
+ * Module file of the TMGMT DGT connector.
+ */
+
+use Drupal\tmgmt_dgt_connector\Notification;
+
+define('TMGMT_DGT_CONNECTOR_MAX_LENGTH', 300);
+define('TMGMT_DGT_CONNECTOR_TRANSLATOR_NAME', 'tmgmt_dgt_connector');
+
+module_load_include('inc', 'tmgmt_dgt_connector');
+
+/**
+ * Implements hook_permission().
+ */
+function tmgmt_dgt_connector_permission() {
+  return array(
+    'administer dgt connector' => array(
+      'title' => 'Administer DGT Connector features',
+      'description' => 'Allows to administer of the DGT Connector features.',
+    ),
+  );
+}
+
+/**
+ * Implements hook_menu().
+ */
+function tmgmt_dgt_connector_menu() {
+  $items = array();
+
+  $items['tmgmt_dgt_connector/server'] = array(
+    'page callback' => '\Drupal\\tmgmt_dgt_connector\\Server::endpoint',
+    // Authentication is done within the service.
+    'access callback' => TRUE,
+  );
+
+  $items['tmgmt_dgt_connector/DGTServicesIntegration.wsdl'] = array(
+    'page callback' => '\Drupal\\tmgmt_dgt_connector\\Server::wsdl',
+    // Authentication is done within the service.
+    'access callback' => TRUE,
+  );
+
+  $items['admin/config/system/dgt_connector'] = array(
+    'title' => 'TMGMT DGT Connector - Configuration',
+    'description' => 'Configuration of the TMGMT DGT Connector.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('tmgmt_dgt_connector_admin_settings_form'),
+    'access arguments' => array('administer dgt connector'),
+    'file' => 'tmgmt_dgt_connector.admin.inc',
+  );
+
+  $items['admin/config/system/dgt_connector/general'] = array(
+    'title' => 'General settings',
+    'type' => MENU_DEFAULT_LOCAL_TASK,
+    'weight' => -10,
+  );
+
+  return $items;
+}
+
+/**
+ * Implements hook_nexteuropa_poetry_notification_translation_received().
+ */
+function tmgmt_dgt_connector_nexteuropa_poetry_notification_translation_received($message) {
+
```

Comment:
Either put them everywhere, either don't put blank lines at the start of a function.